src/riscv-legacy-integration.adoc

```diff
@@ -93,8 +92,7 @@ the sign-extended offset.
 The <<JALR_MODE>> instruction is modal, giving access to the functionality of
 <<JALR>> from either operating mode.
 
-NOTE: <<JALR_MODE>> can be used to change the current CHERI execution mode when
-the implementation supports {cheri_mode_ext_name}.
+NOTE: <<JALR_MODE>> can be used to change the current CHERI execution mode.
```

Comment:
not any more - and I merged the jalr.mode PR so I'm not sure why this section is still here